modules/lambdas/main.tf

```diff
@@ -89,6 +89,34 @@ resource "aws_lambda_function" "delete_old_reconcile_jobs" {
   }
 }
 
+# rule to run the lambda every day
+resource "aws_cloudwatch_event_rule" "delete_old_reconcile_jobs_daily_event_rule" {
+  ## REQUIRED
+  name                = "${var.prefix}_delete_old_reconcile_jobs_daily_event_rule"
+  schedule_expression = "cron(0 5 * * ? *)"  # (Minutes Hours DayOfMonth DayOfWeek Year)
```

Comment:
I think we can skip the weekends as well. Thoughts?

`cron(0 5 ? * MON-FRI *)`